README.md

```diff
@@ -54,12 +55,71 @@ refers to the 10th minute of the concrete hour pointed by the origin. In this ex
 offsets from 0 to 59 corresponding to each one of the 60 minutes within the concrete hour.
 * <b>Samples</b>: For a quadruple range-resolution-origin-offset, it is the number of samples, values, events or notifications available.
 
-###<a id="section1.1"></a> Consuming aggregated time series information
+###<a id="section1.1"></a> Consuming raw data
+
+The STH component exposes an HTTP REST API to let external clients query the raw events (aka. raw data) from which the
+aggregated time series information is generated. A typical URL querying for this information using a GET request is the following:
+
+<pre>http://localhost:8666/STH/v1/contextEntities/type/&lt;entityType&gt;/id/&lt;entityId&gt;/attributes/&lt;attrName&gt;?hLimit=3&hOffset=0&dateFrom=2014-02-14T00:00:00.000Z&dateTo=2014-02-14T23:59:59.999Z</pre>
+
+The entries between "<" and ">" in the URL path depend on the concrete case (type of data, entity and attribute) being queried.
+
+The requests can make use the following query parameters:
+
+* <b>lastN</b>: Only the requested last entries should be returned. It is a mandatory parameter if no hLimit and hOffset are provided.
+* <b>hLimit</b>: In case of pagination, the number of entries per page. It is a mandatory parameter if no lastN is provided.
+* <b>hOffset</b>: In case of pagination, the offset to apply to the requested search of raw data. It is a mandatory parameter if no lastN is provided.
+
+An example response provided by the STH component to a request such as the previous one could be the following:
+
+<pre>
+{
+    "contextResponses": [
+        {
+            "contextElement": {
+                "attributes": [
+                    {
+                        "name": "attrName",
+                        "values": [
+                            {
+                                {
+                                    "timestamp": "2014-02-14T13:43:33.306Z",
```

Comment:
Thanks! ;) Fixed in https://github.com/telefonicaid/IoT-STH/commit/d974bb13ad7cacc770a71cf76f3dfa6fb0a1a20c